controllers/Faq.js

```diff
@@ -48,11 +47,11 @@ faqRouter.post('/insert', function(req, res) {
 });
 
 faqRouter.put('/edit', function(req, res) {
-	var questionToEdit = req.body;
+	const questionToEdit = req.body;
 
 	Faq.findById(questionToEdit.id, function(err, faq) {
-		if (err) return res.send(err);
-		else if (!faq) return res.send('Question ID not found!');
+		if (err) res.status(500).send(err);
```

Comment:
Why did you remove the `return` statement for both cases?

Reply:
My bad. I think I messed up a bit while fixing conflicts